llm/run.sub

```diff
@@ -0,0 +1,37 @@
+JobBatchName            = "LLM training template"
+
```

Comment:
```suggestion
# Update your run name here
arguments               = "demo_run"

```

Moving this to the submit file so that `run.sh` only needs to be modified to disable logging